src/lib.rs

```diff
@@ -89,9 +89,8 @@ pub fn read_strs_to_null<'a>(input: &'a [u8]) -> StrReaderIterator<'a> {
 /// * primitive numeric types (`u8`, `i64`, `f32`, etc.) are fine,
 /// * bools are fine, if the provided data ensures they may have only the values
 ///   `0` or `1` (note that this is a stricter requirement that C),
-/// * structs containing only `Pod` data are fine,
-/// * structs must be `repr(C)` or `repr(packed)`, if the former, the supplied
-///   data must have the correct alignment,
+/// * structs may be `Pod` if they have the `repr(C)` attribute to prevent field
```

Comment:
I think using `repr(packed)` is still valid, since that also preserves ordering.